build/build-report-components.js

```diff
@@ -184,8 +188,17 @@ function makeGenericCreateComponentFunctionCode(compiledTemplates) {
     createFunctionCode('createComponent', lines, ['dom', 'componentName']);
 }
 
+function createReportStylesTemplateEl() {
+  const reportStylesTmplEl = window.document.createElement('template');
+  reportStylesTmplEl.id = 'styles';
+  const reportStyleslEl = window.document.createElement('style');
+  reportStyleslEl.textContent = fs.readFileSync(`${LH_ROOT}/report/assets/styles.css`, 'utf-8');
+  reportStylesTmplEl.content.append(reportStyleslEl);
+  return reportStylesTmplEl;
+}
```

Comment:
Maybe we should grab _all_ the styles from the other components and put them in this one  component. Currently the order of the CSS being applied is defined by the rendering order, which is hard to reason about. It shouldn't be the case that rules defined in one component's stylesheets impact another, but just in case it does we would be better off sorting all the styles by component name and applying them in the same order. And all upfront (not one-at-a-time as encountered)

Reply:
from a quick glance no component css touches anything not defined in its template element (nor should it!) so this is all theoretical. in that case, let's not do this